src/umpire/ResourceManager.cpp

```diff
@@ -77,10 +83,39 @@ ResourceManager::ResourceManager() :
   resource::MemoryResourceRegistry& registry =
     resource::MemoryResourceRegistry::getInstance();
 
+#if defined(UMPIRE_ENABLE_SICM)
+  // read config file/figure out which device belongs to what
+  const std::vector <int> nodes = numa::get_allocatable_nodes();
```

Comment:
FYI I believe using these functions requires `ENABLE_NUMA=ON`.

Reply:
I modified the cmake stuff to build some of the numa stuff when `ENABLE_SICM=ON`, even though `ENABLE_NUMA=OFF`